test/MUXControlsTestApp/MUXControlsTestApp.Shared.projitems

```diff
@@ -37,6 +41,7 @@
     <LangVersion>$(CSLangVersion)</LangVersion>
     <!-- Supress xaml warnings of type foo is defined under contract 'Windows.Foundation.UniversalApiContract' version '4.0.0.0', but the contract version for the targeted min version is '3.0.0.0'! -->
     <SuppressXamlWarnings>0151</SuppressXamlWarnings>
+    <DefineConstants Condition="'$(Platform)' == 'arm64'">$(DefineConstants);ARM64</DefineConstants>
   </PropertyGroup>
```

Comment:
I'm surprised this doesn't come from a common place, but can we at least centralize it in our project to environment.props?

Reply:
don't need it anymore since it is not used anywhere. I'll remove it.